S7.Net/Tcp/TcpClientWrapper.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Net.Sockets;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace S7.Net.Tcp;
+
+public class TcpClientWrapper : TcpClient, ITcpClient

```

Comment:
```suggestion
internal class TcpClientWrapper : TcpClient, ITcpClient

```
Or is it meant to be visible outside of the lib too?